### PR TITLE
Release documentation, automation and website aesthetics

### PR DESCRIPTION
**Pull request by s-canchi**
Added a release plan which lists information on tags and release documentation. Also added the workflow to autogenerate the release notes draft which can be published to the repository when appropriate. Modified the appearance of the heading, footer and added social media icons. 

Preview:
https://cfde-published-documentation--109.com.readthedocs.build/en/109/

### CODE REVIEW

File: Release-Plan/Documentation-Release-Plan.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+This is the release plan for the [CFDE Published Documentation website](https://cfde-published-documentation.readthedocs-hosted.com/en/latest/).
+
+## Release Documentation
+
+The Github repo for the published documentation website will host a release section and associated tag information.
+
+For documentation version numbering system we will follow the release semantics with Major and Minor numbering. Thus our release tag would have the format: **`Major.Minor`**
+
+The current version of the published documentation is at `v1.0`. Any subsequent updates via PRs to the repo will be documented and can be released as either a `Minor` or `Major` release depending on the scope of the content. For example updates to an existing use case might be might be considered a minor update with tag as **`v1.1`**.
```
Comment (ACharbonneau) [inline Release-Plan/Documentation-Release-Plan.md]: to an existing document maybe? not use case

File: mkdocs.yml [hunk 2]
```diff
@@ -1,18 +1,16 @@
...
-    primary: white
+    primary: indigo
   features:
     - navigation.tabs
     - navigation.instant
-  logo: images/CFDE_Logo-color-web.png
-
-docs_dir: docs
-
+  logo: images/CFDE-logo-white-outline.png
```
Comment (ACharbonneau) [inline mkdocs.yml]: can you put the logo back? or change it again, the one you put in doesn't render right

File: Release-Plan/Documentation-Release-Plan.md (hunk 1 above)
Comment (s-canchi) [reply Release-Plan/Documentation-Release-Plan.md]: I have made the corrections.

File: mkdocs.yml (hunk 2 above)
Comment (s-canchi) [reply mkdocs.yml]: I reverted back to the original logo. The white outline logo needs to be redone. The original logo doesn't render well in the smaller browser size in the side nav bar.

**Review by ACharbonneau — APPROVED** [review]

**Comment by ACharbonneau** [comment]
accidentally closed

**Pull request merged**